README.md

```diff
@@ -335,6 +335,23 @@ Schema::create('test', function (Blueprint $table) {
 });
 ```
 
+### Full-text search using FULLTEXT indexes
+
+SingleStoreDB supports full-text search across text columns in a columnstore table using the `FULLTEXT` index type.
+Keep in mind that `FULLTEXT` is not supported when using the `utf8mb4` collation, the table needs to be set to the collation `utf8_unicode_ci`. If you try to add a `FULLTEXT` index to a table with the `utf8mb4` collation an exception will be thrown.
+You can find more information in [the official documentation](https://docs.singlestore.com/managed-service/en/reference/sql-reference/data-definition-language-ddl/create-table.html#fulltext-behavior).
```

Comment:
move link url to end of file

Reply:
Do you mean to the "resources" section? Like this
![Screenshot 2022-11-22 at 8 35 59 PM](https://user-images.githubusercontent.com/6857732/203442400-3c67d5d5-ab59-433c-a4a4-47f6f539362f.jpg)